src/extension/features/budget/subtract-upcoming-from-available/index.js

```diff
@@ -134,30 +145,50 @@ export class SubtractUpcomingFromAvailable extends Feature {
     return totalUpcoming;
   }
 
-  getTotalOfCCBalances(budgetBreakdown) {
-    let totalOfCCBalances = 0;
+  getTotalCCPayments(budgetBreakdown) {
+    let totalCCPayments = 0;
 
     for (const category of budgetBreakdown.inspectorCategories) {
-      if (category.isCreditCardPaymentCategory) totalOfCCBalances -= category.available;
+      if (category.isCreditCardPaymentCategory)
+        totalCCPayments += category.available + category.upcomingTransactions;
     }
 
-    return totalOfCCBalances;
-  }
-
-  getCurrencyClass(amount) {
-    let currencyClass = 'positive';
-
-    if (amount < 0) {
-      currencyClass = 'negative';
-    } else if (amount === 0) {
-      currencyClass = 'zero';
-    }
-
-    return currencyClass;
+    if (totalCCPayments < 0) return 0;
+    return totalCCPayments;
   }
 
   onRouteChanged() {
     if (!this.shouldInvoke()) return;
     this.invoke();
   }
 }
+
+export function createBudgetBreakdownElement(elementId, l10nKey, l10nDefault, amount) {
```

Comment:
I kinda like that you're forcing `l10n` this way but I think it's just fine to pass the `l10n` value in instead so let's just make `title` a parameter here and pass in the evaluated `l10n` string